### ISSUE TITLE
Detection manjaro

### ISSUE THREAD

**Issue by OPortista** [r-darwish/topgrade#155]
I'm on manjaro 18.0.4, kernel 5.0.15
Using the last topgrade 1.12.0 from aur

When I launch topgrade it can't detect the current distribution

**Comment by woohgit** [comment]
I second this. It is not even working from the latest binary upgrade.

```bash
$ topgrade --version
Topgrade 1.12.0

$ topgrade --disable emacs gem vim 

―― Self update ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
Topgrade is up-to-date
Error detecting current distribution: Unknown Linux Distribution

$ cat /etc/lsb-release 
DISTRIB_ID=ManjaroLinux
DISTRIB_RELEASE=18.0.4
DISTRIB_CODENAME=Illyria
DISTRIB_DESCRIPTION="Manjaro Linux"

```

**Comment by r-darwish** [comment]
Should be fixed in 1.13.0
